vendor/src/github.com/docker/distribution/manifest/schema2/manifest.go

```diff
@@ -18,7 +18,7 @@ const (
 	MediaTypeConfig = "application/vnd.docker.container.image.v1+json"
 
 	// MediaTypePluginConfig specifies the mediaType for plugin configuration.
-	MediaTypePluginConfig = "application/vnd.docker.plugin.v0+json"
+	MediaTypePluginConfig = "application/vnd.docker.plugin.image.v0+json"
```

Comment:
The only test plugin used in docker/docker so far is `tiborvass/no-remove` and that has to be repushed. @tiborvass , can you take care of this? Else, we cannot merge this PR since it will result in experimental being red.